sdk/src/Core/Internal/Emitters/UdpSegmentEmitter.cs

```diff
@@ -47,15 +48,17 @@ public UdpSegmentEmitter() : this(new JsonSegmentMarshaller())
         private UdpSegmentEmitter(ISegmentMarshaller marshaller)
         {
             _marshaller = marshaller;
-            _udpClient = new UdpClient();
+            _udpClient = new UdpClient();
             DaemonConfig daemonEndPoint = DaemonConfig.GetEndPoint();
-            EndPoint = daemonEndPoint.UDPEndpoint;
+            _endPoint = daemonEndPoint.UDP_IP_Endpoint;
         }
 
+        private IPEndPoint _endPoint;
+            
         /// <summary>
         /// Gets the end point to daemon.
         /// </summary>
-        public IPEndPoint EndPoint { get; private set; }
+        public IPEndPoint EndPoint => _endPoint;
```

Comment:
We can have the original implementation : ` public IPEndPoint EndPoint { get; private set; }`

Reply:
Addressed in b90e5ee